### PR TITLE
adds compatability with current documentation

### PR DESCRIPTION
**Pull request by enuchi**
Great project. I was able to make some changes so that "make_plugin.js" works (sort of) and downloads the current documentation. There are a couple of new services and some that are deprecated. Also fixes URLs to point to the new documentation. This is working, but throws some errors, so had to "fetch script references" and then build the plug-in separately. You may have better luck on your system.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "tern-gas",
   "version": "0.0.1",
   "description": "A plugin of tern for Google Apps Script.",
-  "main": "gas.js",
+  "main": "gas-2018.js",
```
Comment (aki2o) [inline package.json]: Is this necessary? Do staying`gas.js` have any problem?

File: contrib/fetch_script_reference.js [hunk 2]
```diff
@@ -235,36 +260,53 @@ function parse_type_document(category, typenm, $) {
...
+        return titles.eq(i).parent().parent().find("li");
     }
     console.error("Failed find category element in sidebar : "+category);
     return;
 }
 
+
+
+
+
```
Comment (aki2o) [inline contrib/fetch_script_reference.js]: Is it too much empty rows?

File: contrib/fetch_script_reference.js [hunk 3]
```diff
@@ -90,6 +104,11 @@ function fetch_script_category(category) {
     });
 }
 
+
+/**
+*
+*
+*/
```
Comment (aki2o) [inline contrib/fetch_script_reference.js]: Is this on the way?

File: contrib/fetch_script_reference.js [hunk 4]
```diff
@@ -81,6 +94,7 @@ function is_symbol(str) {
 function fetch_script_category(category) {
     start_task(category);
     var url = get_category_url(category);
+    console.log("category url is : ",url)
```
Comment (aki2o) [inline contrib/fetch_script_reference.js]: Is this for debug? Can you eliminate it?

File: contrib/.tern-project [hunk 5]
```diff
@@ -5,5 +5,6 @@
         "jquery"
     ],
     "plugins": {
+    	"gas-2018": {}
```
Comment (aki2o) [inline contrib/.tern-project]: Is this necessary?

File: contrib/fetch_script_reference.js [hunk 6]
```diff
@@ -1,21 +1,34 @@
 #!/usr/bin/env node
 
 JQUERY_URL = "http://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.js";
-GOOGLE_DOC_SERVER = "http://developers.google.com";
+GOOGLE_DOC_SERVER = "https://developers.google.com";
 ROOT_URI = "/apps-script/reference/";
-SCRIPT_CATEGORIES = ["base", "cache", "charts", "content", "html", "jdbc", "lock", "mail",
+/*
+SCRIPT_CATEGORIES_OLD = ["base", "cache", "charts", "content", "html", "jdbc", "lock", "mail",
```
Comment (aki2o) [inline contrib/fetch_script_reference.js]: I wonder if it is OK to erase it

**Comment by aki2o** [comment]
Thanks a lot! I've commented.

File: package.json (hunk 1 above)
Comment (enuchi) [reply package.json]: No problem, can stay as `gas.js`

File: contrib/fetch_script_reference.js (hunk 2 above)
Comment (enuchi) [reply contrib/fetch_script_reference.js]: yes, can remove

File: contrib/fetch_script_reference.js (hunk 3 above)
Comment (enuchi) [reply contrib/fetch_script_reference.js]: Nope, please remove

File: contrib/fetch_script_reference.js (hunk 4 above)
Comment (enuchi) [reply contrib/fetch_script_reference.js]: yes, can remove

File: contrib/.tern-project (hunk 5 above)
Comment (enuchi) [reply contrib/.tern-project]: I think you can remove

File: contrib/fetch_script_reference.js (hunk 6 above)
Comment (enuchi) [reply contrib/fetch_script_reference.js]: Yes can remove

**Comment by enuchi** [comment]
Appreciate the comments. Sorry for the sloppiness. Let me know if you will merge the changes or if you want me to resubmit.

**Comment by aki2o** [comment]
Would you resubmit at your side?

**Comment by enuchi** [comment]
Cleaned up comments and formatting a bit.

**Comment by aki2o** [comment]
Sorry for being late! Thanks a lot!!!

**Pull request merged**